source/Nuke.Build/INukeBuild.cs

```diff
@@ -19,7 +19,7 @@ public interface INukeBuild
 {
     void ReportSummary(Configure<Dictionary<string, string>> configurator = null);
 
-    internal IReadOnlyCollection<ExecutableTarget> ExecutableTargets { get; }
+    IReadOnlyCollection<ExecutableTarget> ExecutableTargets { get; }
```

Comment:
Should be reverted

Reply:
From my side this change was intentional. To properly be able to override and customize `OnNoTargetsSpecified()` developers need access to the general list of targets which are there as option. 

e.g. in some projects we'd like to only be able to have a "single target" selection as this fits the project setup better and would like to change the default "multi selection" implementation shipped in nuke. 

As alternative I could pass the executableTargets a parameter to `OnNoTargetsSpecified`.

Reply:
From my side this change was intentional. To properly be able to override and customize `OnNoTargetsSpecified()` developers need access to the general list of targets which are there as option. 

e.g. in some projects we'd like to only be able to have a "single target" selection as this fits the project setup better and would like to change the default "multi selection" implementation shipped in nuke. 

As alternative I could pass the executableTargets a parameter to `OnNoTargetsSpecified`.